lib/langchain/llm/ollama.rb

```diff
@@ -18,8 +18,13 @@ class Ollama < Base
 
     # Initialize the Ollama client
     # @param url [String] The URL of the Ollama instance
-    def initialize(url:)
+    # @param completion_model [String] The name of the completion model to use
+    # @param embeddings_model [String] The name of the embeddings model to use
```

Comment:
To adhere to the convention can you rename these 2 to `completion_model_name` and `embeddings_model_name`?